### PR TITLE
Automatic backup

### PR DESCRIPTION
**Pull request by JulianAdams4**

### CODE REVIEW

File: email_mailbox/src/lang/es.json [hunk 1]
```diff
@@ -408,6 +408,57 @@
...
+      "cancelSyncLabel": "Cancelar sincronización"
+    },
+    "manual_sync_device_rejected": {
+      "title": "Autenticación rechazada",
+      "message": "Tu solicitud para acceder a esta cuenta fue rechazada por un dispositivo verificado.",
+      "warning": {
+        "strong": "Advertencia",
+        "text": "Múltiples rechazos podría provocar que el dispositivo sea baneado de Criptext."
+      },
+      "closeButtonLabel": "Okay"
```
Comment (erikaperugachi) [inline email_mailbox/src/lang/es.json]: Aceptar

Comment (erikaperugachi) [inline email_mailbox/src/lang/es.json]: Aceptar

File: electron_app/src/BackupManager.js [hunk 2]
```diff
@@ -155,6 +168,15 @@ const prepareBackupFiles = async ({ backupPrevFiles = true }) => {
   }
 };
 
+const getFilesizeInBytes = filename => {
```
Comment (Hirobreak) [inline electron_app/src/BackupManager.js]: getFile**S**izeInBytes (?)

File: electron_app/src/ipc/backup.js [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+  createDefaultBackupFolder()
+);
+
+const doExportBackupUnencrypted = async params => {
+  const { backupPath, notificationParams } = params;
+  try {
+    globalManager.windowsEvents.disable();
+    commitBackupStatus('local-backup-disable-events', 1);
+    await prepareBackupFiles({});
+    await simulatePause(2000);
```
Comment (Hirobreak) [inline electron_app/src/ipc/backup.js]: why do we need so many pauses?

**Pull request merged**